src/models/group.ts

```diff
@@ -7,16 +7,19 @@ import {
   UpdatedAt,
   DataType,
 } from 'sequelize-typescript'
-import { GroupType } from '../server-internal-types'
+import { GroupType, Location, ContactInfo } from '../server-internal-types'
 
 export interface GroupAttributes {
   id: number
   name: string
   groupType: GroupType
+  primaryLocation: Location
+  primaryContact: ContactInfo
+  website?: string | null
```

Comment:
Suggestion (non-blocking): this translates to `undefined | string | null`. I think `website?: string` should be enough?

Reply:
@deammer it's not enough -- when i had it just as `website?: string` then the linter *sometimes* failed with
`Type 'null' is not assignable to type 'string | undefined'`. 

the only two solutions i found was either this ^^ or pass `website: input.website || undefined` into the create function, and i liked this better. 🤷‍♀️ 

i tried to dig into it, and didn't find a good explanation beyond "graphql scalar types and typescript types sometimes collide because the former is nullable by default and the latter are non-nullable by default"

Reply:
(i would love to find a better solution, that being said. this kinda sucks.)

Reply:
Thanks for looking into it!

Reply:
Thanks for looking into it and the explanation, it seems there are still some rough edges getting sequelize and typescript to cooperate together. I agree that this is better than `website: input.website || undefined` in the create function